Makefile

```diff
@@ -134,17 +143,27 @@ endif
 
 release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-darwin-$(TARGET_ARCH).zip: \
 	release/$(APPLICATION_NAME)-darwin-$(TARGET_ARCH)
-	./scripts/darwin/electron-sign-app.sh -a $</$(APPLICATION_NAME).app -i "$(SIGN_IDENTITY_OSX)"
+ifdef CODESIGN_IDENTITY
+	./scripts/darwin/electron-sign-app.sh -a $</$(APPLICATION_NAME).app -i "$(CODESIGN_IDENTITY)"
+endif
 	./scripts/darwin/electron-installer-app-zip.sh -a $</$(APPLICATION_NAME).app -o $@
 
 release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-darwin-$(TARGET_ARCH).dmg: \
 	release/$(APPLICATION_NAME)-darwin-$(TARGET_ARCH)
+ifdef CODESIGN_IDENTITY
 	./scripts/darwin/electron-installer-dmg.sh -p $< -o $@ \
 		-n "$(APPLICATION_NAME)" \
 		-v "$(APPLICATION_VERSION)" \
-		-d "$(SIGN_IDENTITY_OSX)" \
+		-d "$(CODESIGN_IDENTITY)" \
 		-i assets/icon.icns \
 		-b assets/osx/installer.png
+else
```

Comment:
@lurch I don't know. I thought your addressed this in https://github.com/resin-io/etcher/pull/939#discussion_r90989013